ui/app/AppLayouts/Chat/panels/communities/BackUpCommuntyBannerPanel.qml

```diff
@@ -11,6 +11,7 @@ import "../../popups/community"
 
 Rectangle {
     property var activeCommunity
+    property var store
```

Comment:
hmmm if this is a panel and gets access to the store, should become a view. I know it doesn't do anything else with the store than just passing to the Popup but it could. I would propose:
1) Make it a view (which doesn't make much sense because of passing the store to the popup)
2) Expose a backUpBtnClicked signal and implement its handler in CommunityColumnView

Reply:
Thanks! I was expecting this feedback :D

Gonna go with 2)

Reply:
Updated!